textmining.py

```diff
@@ -0,0 +1,71 @@
+# most common words
+# this code computes the most common words of the adventure_of_sherlock_holmes
+import pickle
+import requests
+import string
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+
+adventure_of_sherlock_holmes = requests.get('http://www.gutenberg.org/ebooks/1661.txt.utf-8').text
+# Analyzing "Adventure of Sherlock Holmes"
+
+# Save data to a file
+f = open('adventure_of_sherlock_holmes.pickle', 'wb')
+pickle.dump(adventure_of_sherlock_holmes, f)
+f.close()
+
+# Load data from a file
+input_file = open('adventure_of_sherlock_holmes.pickle', 'rb')
+reloaded_copy_of_texts = pickle.load(input_file)
+print(type(reloaded_copy_of_texts))
+
+
+def process(s):
+    # creates a dictionary called hist by stripping of the text
```

Comment:
It's good that you included a documentation describing a function, but by convention you put your comments using triple quotes like this:
```
'''
description of function
param: description of parameter
```